bin/visual-regression-test.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Color Variables
+BUILD_DIR=$(pwd)
+txtred=$(tput setaf 1) # Red
+txtgrn=$(tput setaf 2) # Green
+txtylw=$(tput setaf 3) # Yellow
+txtblu=$(tput setaf 4) # Blue
+txtpur=$(tput setaf 5) # Purple
+txtcyn=$(tput setaf 6) # Cyan
+txtwht=$(tput setaf 7) # White
+txtrst=$(tput sgr0) # Text reset.
+
+# Variables
+BUILD_DIR=$(pwd)
+GITHUB_API_URL="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
+
+# Authenticate with Terminus
+./bin/terminus_auth.sh
+
+# Check if we are NOT on the master branch and this is a PR
+if [[ $CIRCLE_BRANCH != "master" && -n "$CI_PULL_REQUEST" ]]
```

Comment:
You could do the reverse and say 

If not {
 exit;
}

continue the rest of the file.

That means we are not indented for the rest of the file by one level